app/controllers/organizations_controller.rb

```diff
@@ -97,6 +112,10 @@ def organization_params
       end
   end
 
+  def destroyable_organization?(organization)
+    organization.organization_memberships.count == 1 && organization.articles.count.zero?
+  end
```

Comment:
I use `.count` here twice, but it might not be necessary.

Reply:
I think it's fine 🤷‍♀️

Reply:
I think so too